src/foundation.cs

```diff
@@ -11058,9 +11064,11 @@ partial interface NSFilePresenter {
 #endif
 
 		[Export ("savePresentedItemChangesWithCompletionHandler:")]
+		[Async]
```

Comment:
right, not sure why it was not spotted (by test) and it should also be an error on the generator -> https://bugzilla.xamarin.com/show_bug.cgi?id=53076